app/celery/reporting_tasks.py

```diff
@@ -142,3 +143,27 @@ def generate_daily_notification_status_csv_report(process_day_string):
 
     current_app.logger.info(f"generate-daily-notification-status-csv-report complete: "
                             f"{len(transit_data)} rows written for day: {process_day}")
+
+
+@notify_celery.task(name="generate-daily-billing-sms-per-use-case-csv-report")
+@statsd(namespace="tasks")
+def generate_daily_billing_sms_per_use_case_csv_report(process_day_string):
+    process_day = datetime.strptime(process_day_string, "%Y-%m-%d").date()
+    transit_data = fetch_sms_billing_per_sms_use_case(process_day)
+    buff = io.StringIO()
+
+    writer = csv.writer(buff, dialect='excel', delimiter=',')
+    header = [
+        "date", "service name", "service id", "channel type" "template name", "template id", "sender", "sender id",
+        "billing code", "count",
+    ]
+    writer.writerow(header)
+    writer.writerows((process_day,) + row for row in transit_data)
+
+    csv_key = f'{process_day_string}.csv'
+    client = boto3.client('s3', endpoint_url=current_app.config['AWS_S3_ENDPOINT_URL'])
+    client.put_object(Body=buff.getvalue(), Bucket=current_app.config['DAILY_STATS_BUCKET_NAME'], Key=csv_key)
+    buff.close()
+
+    current_app.logger.info(f"generate-daily-billing-sms-per-use-case-csv-report complete: "
```

Comment:
For any logs written, please convert to %s string notation (might be worth just going through this entire file for this type of logging).

Reply:
@cris-oddball Just updated. Please check it out

Reply:
It's not the right format. It should replace the variables with "%s". It should look something like this:
`logger.info("complete: %s", var1)`